cookbooks/bcpc/attributes/graphite.rb

```diff
@@ -7,6 +7,7 @@
 default['bcpc']['graphite']['data']['retention'] = 15
 default['bcpc']['graphite']['timezone'] = "'America/New_York'"
 default['bcpc']['graphite']['carbon_fileno_limit'] = 4096
+default['bcpc']['graphite']['install_dir'] = "/opt/graphite"
```

Comment:
If adding an `install_dir` the following should likely be modified to reflect this:
````
  default['bcpc']['graphite']['local_storage_dir'] = "#{node['bcpc'][]'graphite'][;install_dir']}/storage"
 default['bcpc']['graphite']['local_data_dir'] = "#{node['bcpc'][]'graphite'][;install_dir']}/storage/whisper"
 default['bcpc']['graphite']['local_log_dir'] = "#{node['bcpc'][]'graphite'][;install_dir']}/storage/log"
````